packages/tools/pie-icons-vue/.gitignore

```diff
@@ -1 +1,3 @@
 generated/*
+icons/configs-vue.js
```

Comment:
Same question here

Reply:
Same case as [this one](https://github.com/justeattakeaway/pie/pull/386#discussion_r1175066251).